### DISCUSSION TITLE
很适合小白用，但没有交流解答群。QQ群271977815

### DISCUSSION THREAD

**Discussion by laoma2053** [1Panel-dev/1Panel#6828] [Ideas]
很适合小白用，但没有交流解答群。QQ群271977815

尊贵的专业版用户，请进群

**Comment by wanghe-fit2cloud** [comment]
建议多用论坛和官方微信群， 这样有助于开发团队收敛 bug 和需求，否则信息流动不够及时。

[加入我们的微信群，共同推动开源项目发展！](https://bbs.fit2cloud.com/t/topic/2147)

**Reply by laoma2053** [reply]
10元一个月，给不起。是穷逼

**Reply by wanghe-fit2cloud** [reply]
进群后是永久有效的。
